react/src/Auth0.js

```diff
@@ -1,4 +1,4 @@
-import * as Auth0 from "auth0-js";
+import createAuth0Client from '@auth0/auth0-spa-js';
```

Comment:
why are you importing `@auth0/auth0-spa-js` if you installed `@auth0/auth0-react`?  @lguilhermef 

I believe you are following the [Migration Tutorial](https://auth0.com/docs/libraries/auth0-single-page-app-sdk/migrate-from-auth0-js-to-the-auth0-single-page-app-sdk) a bit too full heartedly?

I posted this Migration tutorial to give you some hints where to look, but you are actually not migrating from `auth0-js` to `@auth0/auth0-spa-js`, but from `auth0-js` to `@auth0/auth0-react`. Therefore, I added also the [Auth0 Quickstart Tutorial](https://auth0.com/docs/quickstart/spa/react) for `@auth0/auth0-react` to the ticket.

Both `@auth0/auth0-spa-js` and `@auth0/auth0-react` should be fairly similar, but not 100% the same.